src/hilevel/account.iced

```diff
@@ -247,15 +247,16 @@ exports.Account = class Account
 
   #---------------
 
-  _cpp2_reencrypt_pgp_private_key : ( { me, old_ppc, new_ppc }, cb ) ->
-    output = null
+  _cpp2_reencrypt_pgp_private_keys : ( { me, old_ppc, new_ppc }, cb ) ->
+    outputs = []
     esc = make_esc cb, "_cpp2_reencrypt_pgp_private_key"
-    if (key = me?.private_keys?.primary?.bundle)?
-      await KeyManager.import_from_p3skb { armored : key }, esc defer km
+    for {bundle} in (me?.private_keys?.all or [])
+      await KeyManager.import_from_p3skb { armored : bundle }, esc defer km
```

Comment:
This actually won't work, since we'll be resetting the TripleSec via scrub every time we unlock.